client/blocks/comment-button/index.jsx

```diff
@@ -2,20 +2,26 @@
  * External dependencies
  */
 import React from 'react';
+import { connect } from 'react-redux';
 import noop from 'lodash/noop';
+import classNames from 'classnames';
 
 /**
  * Internal dependencies
  */
 import Gridicon from 'components/gridicon';
+import { getPostTotalCommentsCount } from 'state/comments/selectors';
 
 const CommentButton = React.createClass( {
 
 	propTypes: {
 		onClick: React.PropTypes.func,
 		tagName: React.PropTypes.string,
+		size: React.PropTypes.number,
 		commentCount: React.PropTypes.number,
-		showLabel: React.PropTypes.bool
+		showLabel: React.PropTypes.bool,
+		postId: React.PropTypes.number.isRequired,
```

Comment:
I expect only `CommentButton` to be used in production, which does always require `postId` and `siteId`.

`PureCommentButton` should only be used in tests and dev docs. I'll add a comment noting this in my next commit.